### PR TITLE
Added pypi-tools builder

### PR DESCRIPTION
**Pull request by liiri**

### CODE REVIEW

**Comment by liiri** [comment]
@googlebot I consent.

File: pypi-tools/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Motivation
+
+This cloud-builder can assist with the following use cases:
+
+* Sanity build of a python package
+* Pushing a package to pypi/test.pypi
+* Conditional push of a package - only if a newer version was committed
+
+## Python Version
```
Comment (sanastos) [inline pypi-tools/README.md]: I'm not sure how important this is, but its possible to publish multiple versions with different python versions simultaneously, you can see a similar example here:
https://github.com/GoogleCloudPlatform/cloud-builders-community/blob/master/ng/cloudbuild.yaml

Feel free to ignore if you think one version is sufficient.

File: pypi-tools/cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    id: 'Docker build'
+    args: ['build', '-t', 'gcr.io/$PROJECT_ID/kondunite', '.']
+
+images: ['gcr.io/$PROJECT_ID/kondunite']
```
Comment (sanastos) [inline pypi-tools/cloudbuild.yaml]: Looks like a typo, kondunite -> pypi-tools

File: pypi-tools/examples/cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+steps:
```
Comment (sanastos) [inline pypi-tools/examples/cloudbuild.yaml]: Similar to my comment in your replicated-cli builder PR, can you split this apart into a few named yaml files and add a sanitycheck that verifies the various tools are installed in the default examples/cloudbuild.yaml?

File: pypi-tools/examples/example-package/setup.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+
+from setuptools import setup, find_packages
+
+setup(
+    name='python-package-example',
+    version='0.1',
+    packages=find_packages(exclude=['tests*']),
+    license='MIT',
+    description='An example python package',
+    long_description=open('README.txt').read(),
```
[the remark this answers is not in the document]
Comment (liiri) [reply pypi-tools/examples/example-package/setup.py]: Sorry about that, fixed

**Comment by sanastos** [comment]
Looks good, thanks for the contribution!

**Pull request merged**